.travis.yml

```diff
@@ -53,6 +53,9 @@ addons:
   apt:
     packages:
     - nsis
+    - imagemagick
+    - optipng
```

Comment:
```suggestion

```

No need to install it now :)

Reply:
Bah this is what I get for trying to crunch this in before going somewhere! Fixed.